ocaml/xapi/xapi_host.ml

```diff
@@ -3055,18 +3060,24 @@ let try_restart_device_models_for_recommended_guidances ~__context ~host =
       None
 
 let apply_recommended_guidances ~__context ~self =
```

Comment:
Is it possible to change ~self to ~host? I'm not sure if the API framework allows for it, it would make the code in this functions easier to read